src/main/java/org/openstreetmap/atlas/checks/validation/tag/HighwayToFerryTagCheck.java

```diff
@@ -83,13 +87,19 @@ public boolean validCheckForObject(final AtlasObject object)
         if (Validators.hasValuesFor(object, FerryTag.class))
         {
             final int instructionIndex = hasSameHighwayClassification ? 1 : 0;
-            return Optional.of(this.createFlag(edges,
-                    this.getLocalizedInstruction(instructionIndex, object.getOsmIdentifier())));
+            return Optional.of(this
+                    .createFlag(edges,
+                            this.getLocalizedInstruction(instructionIndex,
+                                    object.getOsmIdentifier()))
+                    .addFixSuggestion(
+                            this.getFixSuggestion(object, true, hasSameHighwayClassification)));
```

Comment:
Multiple Edges can be flagged, but we are only adding a fix suggestion for one of them. A fix suggestion should be added for each edge in `edges`.

Reply:
sorry I missed that.